static/pizza.html

```diff
@@ -0,0 +1,614 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="./bootstrap-3.3.4-dist/css/bootstrap.min.css">
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"></script>
+	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+	<link href="./bootstrap-3.3.4-dist/css/bootstrap.min.css" rel="stylesheet" />
+	<script type="text/javascript" src="https://www.google.com/jsapi"></script>
+	
+    <title>Build Your Pizza</title>
+    
+    <!-- overall content alignment -->  
+    <style>
+    body { background-color: #FFFFFF;}
+        
+	h1 {text-align: center;}
+        
+	.center { 
+        margin-left: auto; 
+        margin-right: auto;}
+        
+    #form span {
+        display: inline-block;
+        vertical-align: middle;
+        margin-left: 40px;}
+
+    .centeral {
+        text-align: center;
+    }
+	</style>
+    
+      
+      
+    <!-- toggle between each of the three builing steps -->
+	<script type="text/javascript">
+	
+	    $(document).ready(function()
+			{
+	               	$('#button1').click(
+	                    function()
+				{
+						document.getElementById("btn1").setAttribute("class","btn btn-primary"); 
+						document.getElementById("btn2").setAttribute("class","btn btn-primary"); 
+	                        $('#size-and-crust').toggle();
+	                        $('#cheese-and-sauce').css({'display':'none'});
+	                        $('#toppings').css({'display':'none'});
+	                        });
+
+	                $('#button2').click(
+	                    function()
+	            {
+	            			 document.getElementById("btn1").setAttribute("class","btn btn-danger"); 
+	            			 document.getElementById("btn2").setAttribute("class","btn btn-primary");  
+	                        $('#size-and-crust').css({'display':'none'});
+	                        $('#cheese-and-sauce').toggle();
+	                        $('#toppings').css({'display':'none'});
+
+	                        });
+	                $('#button3').click(
+	                    function()
+	            {
+	            			document.getElementById("btn2").setAttribute("class","btn btn-danger");  
+	                        $('#cheese-and-sauce').css({'display':'none'});
+	                        $('#toppings').toggle(); 
+	                        $('#size-and-crust').css({'display':'none'});
+	                        });
+	            });
+	</script>
+      
+      
+      
+    <!-- check boxes for choices -->  
+	<script>
+		function chckbx(){
+			//alert("i am here"); 
+			if(document.getElementById('CHEESE').checked)
+		    {
+		    //	alert("chese checked");
+		    document.getElementById('chse').value = 1;
```

Comment:
What you want to do here is build a JSON object that contains the selection, instead of calling document.getElementById('foo').value=1, you just want to do something like
var newPizza = {}
newPizza['cheese'] = document.getElementById('CHEESE').checked
The easiest thing to do is to have a list of properties that gets referenced, rather than having this very long list of if statements. 
After you have constructed the JSON, POST it to the server with jquery per lambdatest.html